scanomatic/image_analysis/image_basics.py

```diff
@@ -59,11 +60,17 @@ def Quick_Scale_To_im(
     scale: Optional[float] = None,
 ) -> np.ndarray:
     if im is None:
+        if path is None:
+            msg = "No image or path supplied"
+            _logger.exception(msg)
+            raise LoadImageError(msg)
+
         try:
             im = load_image_to_numpy(path, dtype=np.uint8)
         except Exception:
-            _logger.error("Could not open source")
-            return -1
+            msg = f"Could not open source with path {path}"
+            _logger.exception(msg)
```

Comment:
```suggestion
            _logger.error(msg)
```
I don't think we need the traceback in the log here